HHS/Head-Start-TTADP
# [TTAHUB-1288] Make sure a screen-reader announces that a form field is required

## Description of change

Handled the various cases where a field was required and wasn't announced as such by the screenreader. react-select needed to be updated to access a required attribute added in a more recent version. Other fields were handled on a case by case basis.

Redid this PR as the original was badly out of sync with main and needed the TR report to be updated as well.

## How to test

- Access the site via a screenreader. I used VoiceOver on Mac, so that's all I 100% vouch for.
- Make sure all required fields in the AR & RTR are announced as required. If an oddball screenreader (the kind you might find in the Chrome webstore) doesn't read off required, let's flag it, but if the appropriate input has the ```required``` attribute, the major screen readers will handle it. ```react-select``` is handled differently, so we'll need to confirm those work as well.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1288


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
